src/controllers/travelDestination.js

```diff
@@ -0,0 +1,27 @@
+import { QueryTypes } from 'sequelize';
+import { sequelize, Location } from '../database/models';
+
+class travelDestination {
+  async getAll(req, res) {
+    const destinations = await sequelize.query(
+      'select destination from "Trips" group by destination order by count(destination) desc',
+      { type: QueryTypes.SELECT }
+    );
+    if (!destinations || !destinations.length) {
+      return res.status(404).json({
+        status: 'failed',
+        msg: res.__('Trips not found')
+      });
+    }
```

Comment:
on the response, is it trips not found or destination not found?
and also as a suggestion you could return success with an empty array instead of a 404

Reply:
I am going to work on it